src/integer_mod_q/z_q/get.rs

```diff
@@ -16,23 +16,50 @@ use crate::{
 impl Zq {
     /// Returns the [`Z`] value of the [`Zq`] element.
     ///
-    /// The representation in the range `[0, modulus[` (`0` inclusive, `modulus` exclusive)
-    /// is returned.
+    /// The representation in the range `[0, modulus)` is returned.
+    /// Use [`Zq::get_representative_around_0`] if they should be
+    /// in the range `[-modulus/2, modulus/2]`.
     ///
     /// # Examples
     /// ```
     /// use qfall_math::integer_mod_q::Zq;
     /// use qfall_math::integer::Z;
     /// let zq_value = Zq::from((4, 7));
     ///
-    /// let z_value = zq_value.get_value();
+    /// let z_value = zq_value.get_representative_0_modulus();
     ///
     /// assert_eq!(Z::from(4), z_value);
     /// ```
-    pub fn get_value(&self) -> Z {
+    pub fn get_representative_0_modulus(&self) -> Z {
         self.value.clone()
     }
 
+    /// Returns the [`Z`] value of the [`Zq`] element with the representatives close to `0`.
+    ///
+    /// The output value is in the range of `[-modulus/2, modulus/2]`.
```

Comment:
How is this interval defined if `modulus` is odd? Which value will be rounded up and which rounded down? Please make this explicite here.